openfermioncirq/primitives/_ffft_test.py

```diff
@@ -0,0 +1,289 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import numpy as np
+import pytest
+
+import cirq
+from cirq import LineQubit
+from openfermioncirq import ffft
+from openfermioncirq.primitives._ffft import (
+    _F0Gate,
+    _TwiddleGate,
+    _inverse,
+    _permute,
+)
+
+
+def _state_from_amplitudes(amplitudes):
```

Comment:
Type annotations. Better names.

I would call this something like a "single electron state" or similar.

Reply:
Done.